Journals_API.ipynb

```diff
@@ -0,0 +1,506 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Journals Search Requests\n",
+        "The ADS maintains a database of summary information about the publications we index -- the Journals database (JournalsDB).  It includes information that the ADS uses for internal content management, but that is also useful for librarians and other content managers to know about how specific titles are classified and what the ADS' holdings are for individual titles and volumes.  This notebook explains how to retrieve information from the JournalsDB using the Journals API.\n",
+        "\n",
+        "The base URL for journals queries is `https://api.adsabs.harvard.edu/v1/journals/<endpoint>`. \n",
+        "\n",
+        "where the allowed `<endpoint>` values are:\n",
+        "* `/Summary/<bibstem>`: generate a summary of information about the publication having `<bibstem>`\n",
+        "* `/Journal/<search text>`:generate a list of bibstems whose journal name and abbreviation data contain `<search text>`; `<search text>` must be url-encoded text\n", 
+        "* `/Refsource/<bibstem>`: generate a list of citation data sources on a per volume basis for a given `<bibstem>`\n",
+        "* `/Holdings/<bibstem>/<volume>`: generate a list of available electronic sources for a given `<bibstem>/<volume>`\n",
+        "\n",
+        "All queries to the API must include an HTTP header that specifies your API access token, e.g.\n",
+        "\n",
+        "`curl -H 'Authorization: Bearer <token>' ...`\n",
+        "\n",
+        "In the examples below, {token_header} is used to denote `-H 'Authorization: Bearer <token>'`, where `<token>` is your personal API access token as described in the top-level [README](https://github.com/adsabs/adsabs-dev-api/blob/master/README.md).\n",
+        "\n",
+        "Notes:\n",
+        "* The information in the JournalsDB is being actively curated and expanded, and records may be added or modified without notice.  For questions about results, please contact adshelp@cfa.harvard.edu.\n",
+        "* All examples use the default HTTP method GET.\n",
+        "* the API examples in this notebook generally assume you're using `curl` statements from the shell command line--you'll see `%%bash` at the top of a cell if its contents should be run in the shell instead of in Python. Guidelines for translating bash `curl` statements into Python can be found in the [converting curl to python notebook](Converting_curl_to_python.ipynb). If you prefer, you can use the unofficial Python package instead; see the top-level [README](https://github.com/adsabs/adsabs-dev-api/blob/master/README.md) for more information."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### `/Summary`\n",
```

Comment:
For all four of these section headers/endpoints, I'd add the required params to the header too, so `/Summary` would be `/Summary/<bibstem>`, just to make it clear what's required of the user

Reply:
Fixed.